tree/ntuple/v7/test/ntuple_processor_join.cxx

```diff
@@ -117,16 +108,14 @@ TEST_F(RNTupleJoinProcessorTest, Basic)
 TEST_F(RNTupleJoinProcessorTest, Aligned)
 {
    try {
-      std::vector<RNTupleOpenSpec> ntuples = {{fNTupleNames[0], fFileNames[0]}, {fNTupleNames[0], fFileNames[0]}};
-      auto proc = RNTupleProcessor::CreateJoin(ntuples, {});
+      auto proc =
+         RNTupleProcessor::CreateJoin({fNTupleNames[0], fFileNames[0]}, {{fNTupleNames[0], fFileNames[0]}}, {});
       FAIL() << "ntuples with the same name cannot be joined horizontally";
    } catch (const ROOT::RException &err) {
-      EXPECT_THAT(err.what(), testing::HasSubstr("horizontal joining of RNTuples with the same name is not allowed"));
+      EXPECT_THAT(err.what(), testing::HasSubstr("joining RNTuples with the same name is not allowed"));
```

Comment:
Is it planned to lift this limitation (and/or is there already an alternative)?  For example in TTree, there is a  syntax to allow setting an alias to allow registration (and distinction) of same name friends.